add info about free space (#1425)
MicrosoftDocs/OfficeDocs-Support: SharePoint/SharePointOnline/lists-and-libraries/url-invalid-upload-file-to-list.md

1. Upload any files that you previously experienced the issue with.

> If the above solution does not solve the issue, please also check that the Sharepoint Database Server is not running out of free space.

Reviewer: ```suggestion
If the previous solution does not solve the issue, also check that the Sharepoint Database Server is not running out of free space.
```

> 1. Go to the SharePoint application server, open Event Viewer and see the details of the actual error. If it's related to database(s) running out of space, proceed to the next step.

Reviewer: ```suggestion
1. Go to the SharePoint application server, open Event Viewer, and see the details of the actual error. If it's related to database(s) running out of space, proceed to the next step.
```

> 1. Check that the following SharePoint databases have enough space to perform actions, and if not, please check the size of the database log and configure the log storage limit accordingly. 

Reviewer: ```suggestion
1. Check that the following SharePoint databases have enough space to perform actions, and if not, check the size of the database log and configure the log storage limit accordingly. 
```

> SharePoing Content Database

Reviewer: ```suggestion
SharePoint content database
```

> SharePoint Contect Database Logs

Reviewer: ```suggestion
SharePoint content database logs
```

> Temp Database Logs

Reviewer: ```suggestion
Temp database logs
```

> SharePoint Service Databases

Reviewer: ```suggestion
SharePoint service databases
```

> SharePoint Service Databases Logs

Reviewer: ```suggestion
SharePoint service databases logs
```

> SharePoint Service Databases Temp Logs

Reviewer: ```suggestion
SharePoint service databases temp logs
```
